jcommune-view/jcommune-web-controller/src/main/java/org/jtalks/jcommune/web/dto/GroupDto.java

```diff
@@ -27,89 +27,64 @@
  *
  */
 public class GroupDto {
+    private static int DISPLAY_USER_LIMIT= 20;
+
     private long id;
     private String name;
+    private List<UserDto> users;
 
-    /**
-     * Instantiates new GroupDto object based on the Group object
-     * @param group source data for the DTO object
-     */
     public GroupDto(Group group) {
         this.id = group.getId();
         this.name = group.getName();
+        this.users = UserDto.convertToUserDtoList(group.getUsers(), UserDto.BY_NAME_COMPARATOR, DISPLAY_USER_LIMIT);
     }
 
-    /**
-     * Instantiates new GroupDto object
-     * @param id id of the group
-     * @param name name of the group
-     */
     public GroupDto(long id, String name) {
         this.id = id;
         this.name = name;
     }
 
-    /**
-     * Default constructor. Needed for initialization from request body.
-     */
     public GroupDto() {
     }
 
-    /**
-     * Gets id of the group
-     * @return id of the group
-     */
     public long getId() {
         return id;
     }
 
-    /**
-     * Sets id of the group
-     * @param id id of the group
-     */
     public void setId(long id) {
         this.id = id;
     }
 
-    /**
-     * Gets name of the group
-     * @return name of the group
-     */
     public String getName() {
         return name;
     }
 
-    /**
-     * Sets name of the group
-     * @param name name of the group
-     */
     public void setName(String name) {
         this.name = name;
     }
 
-    /**
-     * Converts list of the Group objects to the list of the GroupDto objects
-     * @param groups source information about the Groups
-     * @param sortByName if true than result list will be sorted by the group names
-     * @return result list with GroupDto based on the source list of the Group objects
-     */
-    public static List<GroupDto> convertGroupList(List<Group> groups, boolean sortByName) {
-        List<GroupDto> groupDtoList = new ArrayList<GroupDto>();
+    public List<UserDto> getUsers() {
+        return users;
+    }
+
+    public void setUsers(List<UserDto> users) {
+        this.users = users;
+    }
+
+    public static List<GroupDto> convertToGroupDtoList(List<Group> groups, Comparator<GroupDto> comparator) {
+        List<GroupDto> groupDtoList = new ArrayList<>();
+        if (groups == null) {
+            return groupDtoList;
+        }
         for (Group group: groups) {
             groupDtoList.add(new GroupDto(group));
         }
-
-        if (sortByName) {
-            Collections.sort(groupDtoList, BY_NAME_COMPARATOR);
+        if (comparator != null) {
+            Collections.sort(groupDtoList, comparator);
```

Comment:
jcommune stores groups and users in alphabetical case-insensitive order, so you don't need to do additional sorting.